pkg/maps/ctmap/ctmap.go

```diff
@@ -338,7 +338,7 @@ func newMap(mapName string, m mapType) *Map {
 }
 
 func purgeCtEntry6(m *Map, key CtKey, natMap *nat.Map) error {
-	err := m.Delete(key)
+	err := m.SilentDelete(key)
```

Comment:
I think we want to keep `m.Delete(...)` here (same in the v4 case), as these entries (`key`) are read from the CT BPF map, so they should exist (we currently don't support a concurrent CT GC).

Reply:
👍